### PR TITLE
Source Gong: Add new stream `/calls/extensive`

### LINKED ISSUE #23776: Source Gong: add call information stream from /v2/calls/extensive
## Tell us about the problem you're trying to solve
Gong exposes additional information about each call in an endpoint that the connector is currently not using (`/v2/calls/extensive`). There's valuable data in that endpoint, like more info about each call paritcipant.

## Describe the solution you’d like
The ability to pull data about each Gong call participant

Slack thread on this topic: https://airbytehq.slack.com/archives/C021JANJ6TY/p1677812815890109

### PR DESCRIPTION
**Pull request by btkcodedev**
## What
Closes https://github.com/airbytehq/airbyte/issues/23776

## How
Added new stream `extensive calls` 
Docs link:https://us-56804.app.gong.io/settings/api/documentation#post-/v2/calls/extensive

### CODE REVIEW

**Comment by btkcodedev** [comment]
Checked with the builder and verified its working
image

Hash could be seen within manifest :white_check_mark:

**Comment by btkcodedev** [comment]
All checks passed, Ready for review 
CC: @natikgadzhi

File: airbyte-integrations/connectors/source-gong/manifest.yaml
Comment (ChristoGrab) [inline airbyte-integrations/connectors/source-gong/manifest.yaml:805]:
```suggestion
          - integer
```

Oddly enough, even though this value is being extracted from `metadata.id`, which is a `string`, it is being cast as an `integer` in the final record. This might be a quirk of Jinja interpolation? It's not a big enough concern to block this PR, so I'd say let's just update the expected type here and I can create an issue to look into this.

**Review by ChristoGrab — APPROVED** [review]
Great work as per usual @btkcodedev! One small request for a change to the type of the added field, but approving now to unblock you 👍

**Comment by btkcodedev** [comment]
Thanks for your review @ChristoGrab ❤️🎊🎊

**Pull request merged**
